client/components/Event/CreateEventForm/CreateEventFormDumb.js

```diff
@@ -65,6 +64,8 @@ class CreateEventForm extends Component {
             } else {
                 this.props.createEvent(data);
             }
+            this.props.history.push(`/events/${data.id}`)
```

Comment:
I think this should go inside of the else block?

Reply:
fixed